js/controller.js

```diff
@@ -199,6 +211,15 @@
                     $scope.focus = "xkcd";
                 });
             });
+            
+            // Show Dilbert comic
+            AnnyangService.addCommand('Show Dilbert (comic)', function(state, action) {
+                console.debug("Fetching a Dilbert comic for you.");
+                var dilbert = XKCDService.getDilbert("today");  // call it with "random" for random comic
+                $scope.dilbert = dilbert.content;
```

Comment:
Instead of polluting the `$scope` you should bind this to what the service is returning
``` javascript
$scope.dilbert = XKCDService.getDilbert("today");
```
Then in `index.html` you would reference the values like `{{dilbert.title}}` or `ng-src={{dilbert.content}}`